language/compiler/ir_to_bytecode/syntax/src/ast.rs

```diff
@@ -523,6 +526,23 @@ impl Script {
             FunctionBody::Native => panic!("main() can't be native"),
         }
     }
+
+    /// Return a vector of `AccessPath` where the on chain dependent modules are stored.
+    pub fn get_on_chain_deps(&self) -> Vec<AccessPath> {
+        let mut deps = vec![];
```

Comment:
Do we care if there are duplicate imports? We might want to return a set or deduplicate the return vector.